bin/spark-evaluate.js

```diff
@@ -39,7 +39,10 @@ const signer = ethers.Wallet.fromPhrase(WALLET_SEED, provider)
 console.log(
   'Wallet address:',
   signer.address,
-  newDelegatedEthAddress(signer.address, 'f').toString()
+  newDelegatedEthAddress(
+    /** @type {any} */(signer.address),
+    /** @type {import('@glif/filecoin-address').CoinType} */ ('f')
```

Comment:
can we directly pass this instead?

https://github.com/glifio/modules/blob/796cbf39c7e419f62e5c4ddf43f61db2bd14c2bc/packages/filecoin-address/src/coinType.ts#L8

https://github.com/glifio/modules/blob/796cbf39c7e419f62e5c4ddf43f61db2bd14c2bc/packages/filecoin-address/src/index.ts#L9

https://github.com/glifio/modules/blob/796cbf39c7e419f62e5c4ddf43f61db2bd14c2bc/packages/filecoin-address/src/index.ts#L27

```js
coinType.MAIN
```

Reply:
Great suggestion! Applied.